euth/contrib/middleware.py

```diff
@@ -0,0 +1,12 @@
+import pytz
+
+from django.utils import timezone
+
+
+class TimezoneMiddleware(object):
```

Comment:
Extending a class from `object` is the Python2 way to tell the interpreter, that you want new style classes. While it doesn't harm in Python3 it is definitely not needed anymore (see [stackoverflow](http://stackoverflow.com/questions/15374857/should-all-python-classes-extend-object))